### ISSUE TITLE
Allow restricting environmentss the initializers run in

### ISSUE THREAD

**Issue by igorshapiro** [WisePricer/quadro#125]
Here's a possible solution. In `test/quadro.yml`:

```yaml
initializers:
  # Shorthand
  jobs: false
  # Or
  jobs:
    run: false
```

@tomeresk WDYT?

**Comment by noamknispel** [comment]
+1

**Comment by igorshapiro** [comment]
Resolved in commit e8317ca631d8313fc05a2b706caa2b3e897a7fe2
